content/docs/getting-started/create-proxy/authenticated/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to Create an Authenticated Proxy
+publishedDate: '2024-07-18T15:15:00.0Z'
+description: How to create an authenticated proxy in Roadie.
+
+category: proxy-type
+order: 2
+humanName: Authenticated
+scribe: ''
+---
+
+# Create a new Authenticated Proxy
+Roadie provides a custom proxy service that can be configured to proxy through an access token minted by our own authentication system that can be validated by your third party service.
```

Comment:
I'm not sure what this is saying.

Reply:
This is copy/paste from the existing page. I'll see about rewording it.